package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-node",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
doesn't influence anything but conveys the lineage.

Reply:
https://github.com/actions/setup-node/pull/147#pullrequestreview-404993690

Reply:
https://github.com/actions/setup-node/pull/147#pullrequestreview-404993690

Reply:
https://github.com/actions/setup-node/pull/147#pullrequestreview-404993690

Reply:
https://github.com/actions/setup-node/pull/147#pullrequestreview-404993690}https://github.com/Thomasdr06/setup-node/blob/v-mazhuk/automate-releasing-new-versihttps://github.com/Thomasdr06/setup-node/commit/78148dae5052c4942d5b0f92719061df122a3b1cons/tsconfig.json